backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/mail/MailRestController.java

```diff
@@ -237,6 +243,7 @@ private WebuiEmail sendEmail(final WebuiEmail webuiEmail)
 			throw new FillMandatoryException("To");
 		}
 		final EMailAddress to = toList.get(0);
+		final EMailAddress cc = toList.size()>1 ? toList.get(1) : null;
```

Comment:
what happens with toList[2], toList[3] etc?

Reply:
checking at line 260, i see
```java
toList.stream().skip(1).forEach(email::addTo)
```

isn't there an overlap between `to`s and cc?

Reply:
I suggest to 
* either implement support for CC to webui mail (frontend too)
* either accept that Webui Mail does not support CC yet (is that in the requirements?)